docs/_includes/projects/docs/annotations/custom-element.js

```diff
@@ -0,0 +1,44 @@
+import { LitElement, html } from '@polymer/lit-element';
+
+class CustomElement extends LitElement {  
+  static get properties(){
```

Comment:
I think it is more conventional for the opening curly brace is separated from the parentheses by a space:
```js
static get properties() {
```